src/app/app-routing.module.ts

```diff
@@ -58,10 +58,15 @@ const routes: Routes = [
     canActivate: [AuthGuard]
   },
   {
-    path: 'intro',
+    path: '',
```

Comment:
this could cause issues that we saw this morning in dev, instead take a pull my lastest PR has some changes with start url in manifest files there instead of home use intro